### DISCUSSION TITLE
Js调用c再到dart是同步调用吗？

### DISCUSSION THREAD

**Discussion by Owenli0202** [openkraken/kraken#200] [Q&A] [answered]
Js调用c再到dart是同步调用吗？还是类似weex的bridge？

**Comment by andycall** [comment, marked as answer]
是的，在同一个线程进行同步调用

**Reply by Owenli0202** [reply]
非常感谢你的解答，这个同步调用是v8/jsc自身支持的吗？Weex不也是用v8/jsc，为什么不做成同步调用？

**Reply by andycall** [reply]
同步调用是通过 Dart FFI 实现的——直接通过 C 调用 Dart 函数。
